src/AWS.Deploy.CLI/ServerMode/ExtensionMethods.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Security.Claims;
+using Amazon.Runtime;
+
+namespace AWS.Deploy.CLI.ServerMode
+{
+    public static class ExtensionMethods
+    {
+        /// <summary>
+        /// Create an AWSCredentials object from the key information set as claims on the current request's ClaimsPrincipal.
+        /// </summary>
+        /// <param name="user"></param>
+        /// <returns></returns>
```

Comment:
nit: remove empty tag or update it to reflect the return object

Reply:
Done